Detectors/FIT/FT0/reconstruction/include/FT0Reconstruction/CollisionTimeRecoTask.h

```diff
@@ -40,10 +45,25 @@ class CollisionTimeRecoTask
                              gsl::span<const o2::ft0::ChannelData> inChData,
                              gsl::span<o2::ft0::ChannelDataFloat> outChData);
   void FinishTask();
-  // DigitizationParameters const &mParameters;
-  //  o2::ft0::Geometry mGeometry;
+  o2::ft0::FT0ChannelTimeCalibrationObject calibrateTimeOffset(uint32_t orbit, uint16_t bc)
+  {
+    auto& mgr = o2::ccdb::BasicCCDBManager::instance();
+    mgr.setURL("http://ccdb-test.cern.ch:8080");
+    //  getProcessingTimestamp()
+    //    double calib = mCalibApi->getTimeCalibration(dig->getChannel(), dig->getTOT() * Geo::TOTBIN_NS);
+    auto timeStamp = o2::InteractionRecord::bc2ns(bc, orbit);
+    //   mgr.setTimestamp(timeStamp);
+    auto caliboffsets = mgr.get<o2::ft0::FT0ChannelTimeCalibrationObject>("FT0/Calibration/ChannelTimeOffset");
```

Comment:
One should not set-up and query the CCDB for every bc! Do this once per TF in your device run(). Soon the CCDB objects you need will be provided by the DPL processing context.